clang/lib/Driver/ToolChains/Cuda.cpp

```diff
@@ -225,11 +225,7 @@ bool CudaInstallationDetector::validateCudaHeaderDirectory(
         FS.exists(FilePath + "/cuda.h")))
     return false;
   IsIncludePathValid = true;
-  bool IsFound = ParseCudaVersionFile(FilePath + "/cuda.h", Version);
-  if (!IsFound)
```

Comment:
Need keep the code,  orelse the function of validateCudaHeaderDirectory is changed.